### PR TITLE
Add schema_version to output of db info command

### PR DESCRIPTION
**Pull request by kirugan**

### CODE REVIEW

File: cmd/juno/dbcmd.go [hunk 1]
```diff
@@ -97,6 +99,12 @@ func dbInfo(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to get the state update: %v", err)
 	}
 
+	schemaMeta, err := migration.SchemaMetadata(database)
```
Comment (kirugan) [inline cmd/juno/dbcmd.go:102]: maybe we should move it to separate package idk, what is your take on this ?

Comment (weiihann) [reply cmd/juno/dbcmd.go:102]: which package are you looking at? This seems good to me

Comment (kirugan) [reply cmd/juno/dbcmd.go:102]: Something like `db.SchemaMetadata` instead of `migration`

Comment (weiihann) [reply cmd/juno/dbcmd.go:102]: I think migration should have its own package. Perhaps move it to `db` and treat it as a subpackage.

Comment (kirugan) [reply cmd/juno/dbcmd.go:102]: Actually let's keep it as is. I think schema makes sense only for migrations and in this place which is not significant.

**Review by IronGauntlets — APPROVED** [review]

File: cmd/juno/dbcmd.go [hunk 2]
```diff
@@ -97,6 +99,12 @@ func dbInfo(cmd *cobra.Command, args []string) error {
...
+	if err != nil {
+		return fmt.Errorf("failed to get schema metadata: %v", err)
+	}
```
Comment (rodrigo-pino) [inline cmd/juno/dbcmd.go:105]: How much attention should we pay to this warning?

File: cmd/juno/dbcmd.go [hunk 3]
```diff
@@ -97,6 +99,12 @@ func dbInfo(cmd *cobra.Command, args []string) error {
...
+	info.SchemaVersion = schemaMeta.Version
+
```
Comment (rodrigo-pino) [inline cmd/juno/dbcmd.go:107]: Nitpick: is new line (107) intentional?

**Review by rodrigo-pino — APPROVED** [review]

File: cmd/juno/dbcmd.go (hunk 3 above)
Comment (kirugan) [reply cmd/juno/dbcmd.go:107]: not really, let me remove it

File: cmd/juno/dbcmd.go (hunk 2 above)
Comment (kirugan) [reply cmd/juno/dbcmd.go:105]: it's just an error that most likely indicate that there are some

**Pull request merged**